eatin-client/src/components/functions/deleteNewUserRecipe.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback, useState } from "react";
+import { RemoveUserRecipesMutation, useRemoveUserRecipesMutation } from "../../generated/graphql";
+import { useAuth } from "../../context/auth-context";
+
+export function useDeleteUserRecipe() {
+    const [removeSavedUserRecipe] = useRemoveUserRecipesMutation();
+    const [data, setData] = useState<RemoveUserRecipesMutation | null | undefined>(null);
+    const { currentUser } = useAuth();
+
+    const deleteNewUserRecipe = useCallback(
+        (recipeIndex: Number) => {
+            if (currentUser)
```

Comment:
this works good without adding `currentUser` to the hook dependencies?

Reply:
estlint says it needs to be there...